src/main/java/com/example/onlinebookshop/security/SecurityConfig.java

```diff
@@ -0,0 +1,49 @@
+package com.example.onlinebookshop.security;
```

Comment:
```suggestion
package com.example.onlinebookshop.config;
```
replace to another package